spectator-web/src/main/java/com/netflix/spectator/controllers/filter/X.java

```diff
@@ -0,0 +1,7 @@
+package com.netflix.spectator.controllers.filter;
+
+/**
+ * Created by ewiseblatt on 9/6/16.
+ */
+public class X {
```

Comment:
Oops. Forgot to delete this. It was a placeholder class when I created the package to refactor into.